data/json/flags.json

```diff
@@ -1245,6 +1250,11 @@
     "type": "json_flag",
     "context": [  ]
   },
+  {
+    "id": "CRUTCHES",
+    "type": "json_flag",
+    "context": [  ]
+  },
```

Comment:
Duplicate definition?

Reply:
Oh, you are right, no idea how that happened